keylime/secure_mount.py

```diff
@@ -63,7 +63,7 @@ def check_mounted(secdir):
 def mount():
     secdir = os.path.join(config.WORK_DIR, "secure")
 
-    if not config.MOUNT_SECURE:
+    if not config.MOUNT_SECURE or os.getuid != ():
```

Comment:
You mean `or not os.getuid()`? But I am not sure that this is correct. It can be possible that WORK_DIR is already there, `MOUNT_SECURE` as `False`, and after the downgrade the `secure` dir can be created.

If we are creating it as root, that means that we should downgrade early.